docs/releases/1.3.rst

```diff
@@ -0,0 +1,23 @@
+==========================================
+Wagtail 1.3 release notes - IN DEVELOPMENT
```

Comment:
Ok, thanks for the explanation. I thought this PR will at the earliest be merged to master when 1.2 is final.